cmd/goal/bundle_genesis_json.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+THISDIR=$(dirname $0)
+
+cat <<EOM | gofmt > $THISDIR/bundledGenesisInject.go
+// Code generated by bundle_genesis_json.sh, along with langspec.json. DO NOT EDIT.
```

Comment:
I don't think langspec.json is relevant here

Reply:
fixed